content/topics/work/mock-interview/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+ready: true
+title: Mock Interviews
+---
+### Mock Interviews
+
+Mock Interviews are done so give recruits a glimpse of what they can expect from a real
+life interview. This is a great way to prepare them of what is to come.
+
+We have compiled a couple of questions that we know to be likely asked in a real life interview.
+
+This document has those questions and are to be used when conducting a Mock Interview.
+https://docs.google.com/document/d/1O6IGatGF-rUZsMO3ORwVdIigLF4Cgx7-MKsmMQXd4y8/edit
```

Comment:
There are still things missing here. 

What can a recruit expect from a job interview? They wont just be asked canned technical questions.

Interviews often involve:
- Whiteboarding
- code tests (katas)
- technical questions
- talking about experiences and portfolio
- soft skills and culture